superbench/benchmarks/model_benchmarks/pytorch_cnn.py

```diff
@@ -134,8 +134,8 @@ def _inference_step(self, precision):
             self._model.eval()
             while True:
                 for idx, sample in enumerate(self._dataloader):
-                    start = time.time()
                     sample = sample.to(dtype=getattr(torch, precision.value))
```

Comment:
can we just remove this line, and use the corresponding precision (i.e., `self._args.precision`) instead of fp32 when generating dataset? so it's consistent with other transformer models.
https://github.com/microsoft/superbenchmark/blob/ff51a3cee9aa986616bfab4071e9f9ccec81b31b/superbench/benchmarks/model_benchmarks/pytorch_cnn.py#L57